.github/workflows/destroy_aks.yml

```diff
@@ -0,0 +1,46 @@
+name: Destroy Review Instance
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  destroy:
+    name: Destroy
+    environment:
+       name: review_aks
+    runs-on: ubuntu-latest
+    concurrency: review_aks${{github.event.number}}
+
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set Environment variables
+        id: set_env_var
+        shell: bash
+        run: |
+          tf_vars_file=terraform/aks/config/${{ inputs.environment }}.tfvars.json
```

Comment:
${{ inputs.... }} is only valid inside an action, not a workflow. This can be hardcoded to "review" here

Reply:
done